web/src/js/components/MainView.tsx

```diff
@@ -6,11 +6,12 @@ import {useAppSelector} from "../ducks";
 
 export default function MainView() {
     const hasSelection = useAppSelector(state => !!state.flows.byId[state.flows.selected[0]])
+    const connection_established = useAppSelector(state => state.connection.state) === 'CONNECTION_ESTABLISHED'
     return (
         <div className="main-view">
             <FlowTable/>
-            {hasSelection && <Splitter key="splitter"/>}
-            {hasSelection && <FlowView key="flowDetails"/>}
+            {(hasSelection && connection_established) && <Splitter key="splitter"/>}
+            {(hasSelection && connection_established) && <FlowView key="flowDetails"/>}
```

Comment:
This shouldn't be a requirement here. (Why) is this necessary?

Reply:
There is a bug without this: 
1. Open mitmweb and record some request.
2. Change the `content_view_lines_cutoff` from default to another value
3. Click on a request  to open the flow details.
4. Refresh page.

Result:
It is possible for `/flows/<flow_id>/request/content/Auto.json?lines=<content_view_lines_cutoff>` to start before `/options` gets response. In this case the `content_view_lines_cutoff` will use the default value.

Reply:
I see, thanks. That's not ideal, but also not a dealbreaker. We don't want to make flow display conditional on having a connection though, so let's just accept this edge case and remove the special case here. :) We want to be able to show flows even if the connection is lost.

Reply:
Sure, removed.